src/pages/Home/index.js

```diff
@@ -1,19 +1,59 @@
-import React from "react";
-import { Banner, Products } from "../../components";
+import React, { useMemo } from "react";
+import { Banner, Products, Loader } from "../../components";
+import { useRequests } from "../../hooks";
 
-import featuredBanners from "../../mocks/en-us/featured-banners.json";
-import productCategories from "../../mocks/en-us/product-categories.json";
-import featuredProducts from "../../mocks/en-us/featured-products.json";
+function Home() {
+  const requests = useMemo(
+    () => [
+      {
+        name: "featuredBanners",
+        params: {
+          q: '[[at(document.type, "banner")]]',
+          pageSize: 5,
+        },
```

Comment:
I believe we reviewed this code during our 1:1. Not sure if you explore the possibility of creating this via function. It would help with the reading cause you would end up doing something like:

```javascript
=> [
   createRequest('featuredBanners', 'product', 5),
   createRequest('productCategories', 'category', 30),
...
]
```

Reply:
Also consider moving this array to your constants file (regardless of the approach), as these queries won't change.